packages/jss-vendor-prefixer/src/index.test.js

```diff
@@ -0,0 +1,165 @@
+import expect from 'expect.js'
+import {create} from 'jss'
+import cssVendor from 'css-vendor'
+import browser from 'detect-browser'
+import functionPlugin from 'jss-plugin-syntax-rule-value-function'
+
+import vendorPrefixer from './index'
+
+const settings = {
+  createGenerateClassName: () => rule => `${rule.key}-id`
+}
+
+const isIE9 = browser.name === 'ie' && browser.version === '9.0.0'
+const isIEorEdge = browser.name === 'edge' || browser.name === 'ie'
+
+describe('jss-vendor-prefixer', () => {
+  let jss
+
+  beforeEach(() => {
+    jss = create(settings).use(vendorPrefixer())
+  })
+
+  describe('prefixed property', () => {
+    if (isIE9) {
+      return
+    }
+
+    let sheet
+
+    beforeEach(() => {
+      sheet = jss.createStyleSheet({
+        a: {animation: 'yyy'}
+      })
+    })
+
+    it('should generate correct CSS', () => {
+      if (isIEorEdge) {
+        expect(sheet.toString()).to.be('@keyframes a {\n}')
+        return
+      }
+      const prefixedProp = cssVendor.supportedProperty('animation')
+      expect(sheet.toString()).to.be(`.a-id {\n  ${prefixedProp}: yyy;\n}`)
+    })
+  })
+
+  describe('@keyframes', () => {
+    let sheet
+
+    beforeEach(() => {
+      sheet = jss.createStyleSheet({
+        '@keyframes a': {}
+      })
+    })
+
+    it('should generate correct CSS', () => {
```

Comment:
Also, this one fails in the BrowserStack this test fails as well.

Reply:
Apparently, the keyframes aren't being prefixed in IE 9.0.0 (Windows 7.0.0), IE 10.0.0 (Windows 8.0.0), IE 11.0.0 (Windows 8.1.0.0), Edge 15.15063.0 (Windows 10.0.0)

Reply:
So they assertion is wrong.

Reply:
the assertion is not running, see early exit before on ln 24

Reply:
wait, early exit is only IE9, others should not add any prefixes afaik, cc @AleshaOleg how did the tests even pass?

Reply:
That's a different test. Two tests are failing right now

Reply:
I think the jss-vendor-prefixer plugin wasn't tested with BrowserStack and locally those tests succeed.

Reply:
Ohhh I know whats going on here...

The check for IE and Edge is supposed to be in this test and not in the other test.

That way the tests make sense...

Reply:
@kof tests seem to pass now